buildSrc/src/main/kotlin/embrace-defaults.gradle.kts

```diff
@@ -85,6 +85,7 @@ android {
 }
 
 dependencies {
+    implementation("androidx.startup:startup-runtime:1.1.1")
```

Comment:
Is there a way we can get versions in this into a toml? Having that and `Versions.kt` is kind of annoying...

Reply:
Unfortunately not that I'm aware of. This is a deficiency of gradle - this script is effectively precompiled at the same time as the version catalog, so it's not possible to use the version catalog as there would be a circular dependency. Hopefully this oversight is fixed in newer versions of Gradle.